src/main/java/hudson/tasks/test/AbstractTestResultAction.java

```diff
@@ -26,31 +26,40 @@
 import hudson.Extension;
 import hudson.Functions;
 import hudson.model.*;
+import hudson.tasks.junit.CaseResult;
+import hudson.tasks.junit.SuiteResult;
 import hudson.util.*;
 import hudson.util.ChartUtil.NumberOnlyBuildLabel;
 
 import java.awt.*;
 import java.io.IOException;
-import java.util.Collections;
+import java.util.*;
```

Comment:
Its not a good idea to do `import *`. This has several disadvantages, major one being if some later version of java add an other interface like `java.util.chart` and if there is any existing usage of `chart` from some other lib, there'll be conflict and code might as well break.
Please refrain from using `import *` and import specific packages being used.

Reply:
Ok,Thanks. I will make the required changes and will keep that in mind for future.